rapid7_insightvm/komand_rapid7_insightvm/util/resource_helpers.py

```diff
@@ -0,0 +1,182 @@
+import defusedxml.ElementTree as etree
+import urllib3
+import re
+from .exceptions import ResourceNotFound
+from . import endpoints, resource_requests
+
+
+class V1Session:
+    """
+    Used to authenticate and deauthenticate from the v1 API
+    """
+
+    def __init__(self, session, logger):
+        """
+        Creates a new instance of V1Session
+        :param session: Session object available to Komand actions/triggers, usually self.connection.session
+        :param logger: Logger object available to Komand actions/triggers, usually self.logger
+        :return: ResourceHelper object
+        """
+        self.logger = logger
+        self.session = session
+
+        # Suppress insecure request messages
+        urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
```

Comment:
You may want to move this to the top of the file but beneath the imports so it's only executed once